airbyte-commons/build.gradle

```diff
@@ -3,6 +3,5 @@ plugins {
 }
 
 dependencies {
-    implementation 'org.apache.commons:commons-compress:1.20'
-    implementation 'org.apache.commons:commons-lang3:3.11'
+    // Dependencies for this module should be specified in the top-level build.gradle.
```

Comment:
why?

Reply:
Because of https://github.com/airbytehq/airbyte/blob/master/build.gradle#L181, if everything is depending on airbyte-commons, it seems cleaner to have all of common's dependencies in the build.gradle as well.